docs/guide/dev/integration/angular/README.md

```diff
@@ -0,0 +1,245 @@
+
+### `editorUrl`
+see [Customising CKEditor Preset or Version](#customizing-ckeditor-preset-or-version)
+
+### `config`
+
+Custom configuration can be passed to the editor via `config` attribute passed to `<ckeditor>` in component template. The following example shows {@link guide/dev/features/toolbar/README how to change the contents of the toolbar}:
+
+```html
+<ckeditor
+	data="<p>Editor' content</p>"
+	[config]="{ { toolbar: [ [ 'Bold' ] ] } }"
+></ckeditor>
+```
+
+{@linkapi CKEDITOR.config All configuration} options can be changed this way.
+
+### `data`
+
+The initial data of the editor. It can be a static value:
+
+```html
+<ckeditor data="<p>Hello, world!</p>" ...></ckeditor>
+```
+
+or a shared parent component’s property
+
+```typescript
+@Component( {
+    ...
+} )
+export class MyComponent {
+    public editorData = '<p>Hello, world!</p>';
+    ...
+}
+```
+
+```html
+<ckeditor [data]="editorData" ...></ckeditor>
+```
+
+### `tagName`
+
+Specifies the tag name of the HTML element on which the editor will be created.
+
+The default tag is `<textArea>`.
+``` html
+<ckeditor tagName="textarea" ...></ckeditor>
+```
+
+### `readOnly`
+
+Editor can be set to {@link guide/dev/features/readonly/README read-only mode} via `readOnly` property:
+
+```html
+<ckeditor
+	data="<p>Editor's content</p>"
+	readOnly="true"
+><ckeditor>
+```
+
+This property takes precedence over {@linkapi CKEDITOR.config#readOnly `config.readOnly`} setting.
+
+## Supported `@Output` properties
+
+The following `@Input` properties are supported by CKEditor 4 Angular component:
+
+### `ready`
+
+Fires when the editor is ready. It corresponds with the `editor#instanceReady` https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#event-instanceReady event.
+
+### `change`
+
+Fires when the content of the editor has changed. In contrast to `change` - only emits when data really changed thus can be successfully used with `[data]` and two way `[(data)]` binding.
+
+Fires when the content of the editor has changed. It corresponds with the `editor#change` https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#event-change event. For performance reasons this event may be called even when data didn't really changed.
+
+```html
+<ckeditor (change)="onChange($event)"></ckeditor>
+```
+
+``` typescript
+
+import { CKEditor4 } from "@ckeditor/ckeditor4-angular/ckeditor";
+
+@Component( {
+    ...
+} )
+export class MyComponent {
+    public onChange( event: CKEditor4.EventInfo ) {
+        console.log( event.editor.getData() );
+    }
+    ...
+}
+```
+
+### `dataChange`
+
+Fires when the content of the editor has changed. In contrast to `change` - only emits when data really changed thus can be successfully used with `[data]` and two way `[(data)]` binding.
+
+### `focus`
+
+Fires when the editing view of the editor is focused. It corresponds with the `editor#focus` https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#event-focus event.
```

Comment:
Most common name for this is editable, let's stick to it.

```suggestion
Fires when the editor's editable is focused. It corresponds with the `editor#focus` https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#event-focus event.
```

Same change for the blur event.

Reply:
It should be a link, instead there's a event name followed by a full URL. Same thing for blur.